libs/sdk-core/src/breez_services.rs

```diff
@@ -1042,6 +1042,14 @@ impl BreezServices {
         Ok(self.node_api.execute_command(command).await?)
     }
 
+    // Collects various user data from the node and the sdk storage.
+    // This is used for debugging and support purposes only.
+    pub async fn collect_user_data(&self) -> SdkResult<String> {
```

Comment:
I would avoid calling it `collect_user_data`, especially after reading the Apple privacy manifest guide. Maybe something like `get_diagnostic_data`

Reply:
Done